### ISSUE TITLE
previews

### ISSUE THREAD

**Issue by ewoks** [PaulWay/openscad_work#1]
just as an idea: 
would it make sense to add mini photo of the printed model, so others get idea before printing? 
An image (360p or 480p) file next to the model should be enough, so it it is previewed in a browser.

Thanks for consideration

**Comment by PaulWay** [comment]
Hi @ewoks - yep, that's a good idea!  Certainly a better Readme would be good, explaining what the models are and what the modules in the `pw_primitives.scad` do.

Do you know if it's possible to give OpenSCAD a scene description (using the `$vpr`, `$vpt`, `$vpf` and `$vpd` variables, say) and get it to generate a scene into a `.png` file from the command line?  I'd like to automate this if I can :-)

**Comment by PaulWay** [comment]
(Not that I'm putting that research on you, just asking...)

**Comment by ewoks** [comment]
not really, I don't know that :( I'm just getting into whole 3d printing world
